aries-site/src/data/wcag/tag.json

```diff
@@ -0,0 +1,518 @@
+[
+  {
+    "rule": "1.1.1",
+    "label": "Non-text content",
+    "status": "passed",
+    "notes": "There is an aria-label on the close icon button but no way for the implementer to choose their own."
+  },
+  {
+    "rule": "1.2.1",
+    "label": "Audio-only and Video-only (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.2",
+    "label": "Captions (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.3",
+    "label": "Audio Description or Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.4",
+    "label": "Captions (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.5",
+    "label": "Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.6",
+    "label": "Sign Language (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.7",
+    "label": "Extended Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.2",
+    "label": "Meaningful Sequence",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.3",
+    "label": "Sensory Characteristics",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.4",
+    "label": "Orientation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.5",
+    "label": "Identify Input Purpose",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.6",
+    "label": "Identify Purpose",
+    "status": "failed",
+    "notes": "The close icon doesn't have a descriptive aria-label"
```

Comment:
I dropped a comment on this previously, the comment/any replies might've gotten lost in the refactor but I would say this passes. The svg has "Close" on it. Feels like we can enhance to allow more of a customization but would that be sufficient?

Reply:
Agree, I'll update this to passed